client/src/components/participant-details/psi-cohort.js

```diff
@@ -100,7 +100,23 @@ const PSIRouteTabs = ({
           render={() => <h1>Work in progress</h1>}
         ></Route>
         <Route exact path={TabDetails.assignCohort.path}>
-          <PSICohortTable disabled={disabled} rows={psiList} assignAction={assignAction} />
+          {disabled && (
+            <div>
+              <Box>
+                <Typography variant='h3'> Assigning Cohort</Typography>
+                <br />
+                <Typography variant='body1'>
+                  {' '}
+                  This participant has been assigned cohort. To re-assign cohort. Please click on
+                  withdraw cohort to withdraw participant from cohort. After that, you can assign
+                  cohort in "Assigning Cohort"
```

Comment:
Suggested Copy Change:
This participant has already been assigned a cohort. To reassign the cohort, please click on 'Withdraw Cohort' to withdraw them first.